kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -98,6 +107,108 @@ public static Map<String, Map<String, Variant<?>>> buildSettings(NetworkProperti
         return newConnectionSettings;
     }
 
+    public static Map<String, Variant<?>> build8021xSettings(NetworkProperties props, String deviceId) {
+
+        String eap = props.get(String.class, "net.interface.%s.config.802-1x.eap", deviceId);
```

Comment:
One question: NetworkManager actually supports `eap` as a list of strings since it can support multiple auth methods concurrently. Here you're only expecting one string. Is this a design constraint we chose to implement?

What happens if, one day, we decide to support multiple auths? Will this break snapshots?